docs/advanced/refs.md

```diff
@@ -1,7 +1,59 @@
 # Refs
 *Refs* grant access to the actual Instance objects that are created by Roact. They're an escape hatch for when something is difficult or impossible to correctly express with the Roact API.
 
-To create a ref, pass a function prop with the key `Roact.Ref` when creating a primitive element:
+!!! info
+	Refs can only be used with primitive components.
+
+## Refs in Action
+
```

Comment:
We should try to be consistent about whether we have blank lines after each Markdown header. I err on the side of not having a newline, which visually groups sections a little better.